CHANGELOG.md

```diff
@@ -38,6 +38,10 @@
  * Note that `email_verified == not email_verification_required`.
  * A data migration will be necessary.
 
+## v8.1.1
+
+* Bugfix: Don't show "passwords do not match" when the first password is invalid.
```

Comment:
Actually -- I got the bisect the wrong way round.. Oops!

![https://raw.githubusercontent.com/meshy/gifs/master/picard-giggle.gif](https://raw.githubusercontent.com/meshy/gifs/master/picard-giggle.gif)